### PR TITLE
refactor: Removed unused classes

### PR DESCRIPTION
**Pull request by Balcan**

### CODE REVIEW

File: app/src/main/java/org/dhis2/usescases/teiDashboard/dashboardfragments/tei_data/TEIDataPresenterImpl.java [hunk 1]
```diff
@@ -256,16 +251,7 @@ public void onShareClick(View mView) {
...
-        extras.putString("PROGRAM_UID", programUid);
-        if (dashboardProgramModel.getCurrentEnrollment() != null)
-            extras.putString("ENROLLMENT_UID", dashboardProgramModel.getCurrentEnrollment().uid());
-        intent.putExtras(extras);
         ActivityOptionsCompat options = ActivityOptionsCompat.makeSceneTransitionAnimation(view.getAbstractActivity(), sharedView, "user_info");
-//        view.seeDetails(intent, options.toBundle());
-
         view.seeDetails(EnrollmentActivity.Companion.getIntent(view.getContext(),
                 dashboardProgramModel.getCurrentEnrollment().uid(),
                 dashboardProgramModel.getCurrentProgram().uid(),
```
Comment (ferdyrod) [inline app/src/main/java/org/dhis2/usescases/teiDashboard/dashboardfragments/tei_data/TEIDataPresenterImpl.java]: We should look for a way to move this calls (Intent and ActivityOptionsCompat) to the view. As well as removing View parameter from the method

File: app/src/main/java/org/dhis2/usescases/teiDashboard/dashboardfragments/tei_data/TEIDataPresenterImpl.java [hunk 2]
```diff
@@ -274,18 +260,7 @@ public void seeDetails(View sharedView, DashboardProgramModel dashboardProgramMo
...
         Intent intent = ScheduledEventActivity.Companion.getIntent(view.getContext(),uid);
-        /*Intent intent = new Intent(view.getContext(), EventDetailActivity.class);
-        Bundle extras = new Bundle();
-        extras.putString("EVENT_UID", uid);
-        extras.putString("TOOLBAR_TITLE", title);
-        extras.putString("TEI_UID", teiUid);
-        intent.putExtras(extras);*/
         ActivityOptionsCompat options = ActivityOptionsCompat.makeSceneTransitionAnimation(view.getAbstractActivity(), sharedView, "shared_view");
         view.openEventDetails(intent, options.toBundle());
     }
```
Comment (ferdyrod) [inline app/src/main/java/org/dhis2/usescases/teiDashboard/dashboardfragments/tei_data/TEIDataPresenterImpl.java]: Same as previous comment

Comment (Balcan) [reply app/src/main/java/org/dhis2/usescases/teiDashboard/dashboardfragments/tei_data/TEIDataPresenterImpl.java]: Agreed, but not for now.

Comment (ferdyrod) [reply app/src/main/java/org/dhis2/usescases/teiDashboard/dashboardfragments/tei_data/TEIDataPresenterImpl.java]: ok

**Pull request merged**